README-REST.md

```diff
@@ -0,0 +1,1122 @@
+  * [Time](#get-time)
+  * [Markets](https://github.com/bitvavo/python-bitvavo-api#get-markets)
+  * [Assets](https://github.com/bitvavo/python-bitvavo-api#get-assets)
+* [Public market data](#public-market-data)
+  * [Book](https://github.com/bitvavo/python-bitvavo-api#get-book-per-market)
+  * [Public Trades](https://github.com/bitvavo/python-bitvavo-api#get-trades-per-market)
+  * [Candles](https://github.com/bitvavo/python-bitvavo-api#get-candles-per-market)
+  * [Price Ticker](https://github.com/bitvavo/python-bitvavo-api#get-price-ticker)
+  * [Book Ticker](https://github.com/bitvavo/python-bitvavo-api#get-book-ticker)
+  * [24 Hour Ticker](https://github.com/bitvavo/python-bitvavo-api#get-24-hour-ticker)
+* [Private trading data](#private-trading-data)
+  * [Place Order](https://github.com/bitvavo/python-bitvavo-api#place-order)
+  * [Update Order](https://github.com/bitvavo/python-bitvavo-api#update-order)
+  * [Get Order](https://github.com/bitvavo/python-bitvavo-api#get-order)
+  * [Cancel Order](https://github.com/bitvavo/python-bitvavo-api#cancel-order)
+  * [Get Orders](https://github.com/bitvavo/python-bitvavo-api#get-orders)
+  * [Cancel Orders](https://github.com/bitvavo/python-bitvavo-api#cancel-orders)
+  * [Orders Open](https://github.com/bitvavo/python-bitvavo-api#get-orders-open)
+  * [Trades](https://github.com/bitvavo/python-bitvavo-api#get-trades)
+  * [Account](https://github.com/bitvavo/python-bitvavo-api#get-account)
+  * [Balance](https://github.com/bitvavo/python-bitvavo-api#get-balance)
+  * [Deposit Assets](https://github.com/bitvavo/python-bitvavo-api#deposit-assets)
+  * [Withdraw Assets](https://github.com/bitvavo/python-bitvavo-api#withdraw-assets)
+  * [Deposit History](https://github.com/bitvavo/python-bitvavo-api#get-deposit-history)
+  * [Withdrawal History](https://github.com/bitvavo/python-bitvavo-api#get-withdrawal-history)
+
+### General
+
+#### Rate limiting
+
+Bitvavo uses a weight based rate limiting system. Your app is limited to 1000 weight points per IP or API key per 
+minute. When you make a call to Bitvavo API, your remaining weight points are returned in the header of each REST request. 
+
+Websocket functions do not return your returning weight points, you track your remaining weight points with a call to:
+```
+limit = bitvavo.getRemainingLimit()
+```
+
+If you make more requests than permitted by the weight limit, your IP or API key is banned. 
+
+The rate weighting for each endpoint is supplied in the [Bitvavo API documentation](https://docs.bitvavo.com/).
+
+#### REST requests
+
+For all functions, required parameters are passed as separate values, optional parameters are passed as a dictionary; 
+return parameters are in dictionary format such that `response['<key>'] = '<value>'`.  Only when [placing orders](https://github.com/bitvavo/python-bitvavo-api#place-order) some of the optional parameters are required, since a limit order requires more information than a market order. 
```

Comment:
Unnecessary fully-qualified URL
```suggestion
return parameters are in dictionary format such that `response['<key>'] = '<value>'`.  Only when [placing orders](#place-order) some of the optional parameters are required, since a limit order requires more information than a market order. 
```

Reply:
Fixed